portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,45 +14,77 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.data.Message;
 import java.io.IOException;
 import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.gson.Gson;
 
 /*
-*  Servlet that adds comments
-*/
+ *  Servlet that adds comments
+ */
 @WebServlet("/handle-comments")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages;
-
   @Override
-  public void init() {
-    messages = new ArrayList<>();
-  }
-  
-  
+  public void init() {}
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("add-content");
-    messages.add(text);
+    String color = request.getParameter("color");
+    String font_size = request.getParameter("font_size");
+    long time = System.currentTimeMillis();
+    if (text == "") {
+      response.sendRedirect("comment.html");
```

Comment:
I don't think you need to `sendRedirect` here. Since no comments are being added, there are no new comments that need displaying, and therefore no need to reload the page.